src/main/java/http/helper/Handler.java

```diff
@@ -0,0 +1,9 @@
+package http.helper;
+
+import http.request.Request;
+import http.response.Response;
+
+@FunctionalInterface
```

Comment:
TIL - I'm seeing this for the first time, thank you!

Reply:
I learnt it from Sam's code.